package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dashevo/dp-services-ctl",
-  "version": "0.7.4",
+  "version": "0.8.0",
```

Comment:
I think we need to create dev branch `v0.8-dev` and use `0.8.0-dev.1` version here.
But do not forget to update Travis build such as we did in `js-dpp` and `drive`.